src/linear_algebra/basic.lean

```diff
@@ -52,6 +52,10 @@ We note that, when constructing linear maps, it is convenient to use operations
 maps (`prod`, `coprod`, arithmetic operations like `+`) instead of defining a function and proving
 it is linear.
 
+`linear_map.prod` and `linear_map.coprod` are `linear_equiv`s in `S` between `linear_map`s in
+`R`. When we have `comm_semiring R`, `S = R` can be used (via `nat_smul_comm_class`). When we do
```

Comment:
We don't need `nat_smul_comm_class` in case `S = R`.

Reply:
Whoops, typo:
```suggestion
`R`. When we have `comm_semiring R`, `S = R` can be used (via `smul_comm_class_self`). When we do
```